_maps/configs/inteq_vaquero.json

```diff
@@ -0,0 +1,47 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"map_name": "Vaquero-Class Light Frigate",
```

Comment:
not sold on the name honestly. Very Solar flavoring. The lorbs might have some animal (perhaps a kalixian draft beast) that fits better.